src/libman/Resources/Text.resx

```diff
@@ -132,6 +132,9 @@
   <data name="DefaultProviderOptionDesc" xml:space="preserve">
     <value>The provider to use if no provider is defined for a given library. (eg. “cdnjs”, “filesystem”)</value>
   </data>
+  <data name="NonInteractiveConsoleMessage" xml:space="preserve">
+    <value>Cannot start an interactive session. Please specify all required arguments on the commandline.</value>
```

Comment:
This message is not actionable: what are the arguments I have to specify?  I can see this being a frustrating user experience.

Can you add a list of arguments?  Or I'd even be fine if you rephrased to the one argument at a time, which might be easier depending on how the command is processed, e.g. "Cannot start an interactive session. Please specify a value for the {--foo} argument on the command line."  The user would have to iterate if multiple were required, but each one would be clearly actionable.

Reply:
This message is shown after `ShowHelp()` which will show the arguments and options for the command being run.

Reply:
It doesn't seem clear to me from `libman init --help` which parameters are required (mostly because none of them are required).  I don't think relying upon `ShowHelp()` gives the best experience here.

Reply:
What if it was "Please specify all arguments on the command line"? Does that read as "all the arguments you want to specify", or does it make you think that every single argument is required whether you want it or not?

---
In reply to: [242353043](https://github.com/aspnet/LibraryManager/pull/410#discussion_r242353043) [](ancestors = 242353043)

Reply:
Changed the messaging to show the field name and option string.

For instance on running libman init the user will see the below: 
```
Cannot prompt for user input in this console.
Please specify 'DefaultProvider' using the '--default-provider' option.
```